packages/styles/components/_c.select.scss

```diff
@@ -0,0 +1,96 @@
+$select-border-radius : get-border-radius('m');
+$select-border-width : get-border('s');
+
+.mc-select {
+  border: $select-border-width solid $color-select-border;
+  border-radius: $select-border-radius;
+  display: block;
+
+  @include set-font-scale('05');
+
+  background: $color-select-background;
+  color: $color-select-text;
+  position: relative;
+  width: 100%;
+
+  /* autoprefixer: ignore next */
+  -webkit-transform: translateZ(0);
```

Comment:
why ? add a comment to explain why you translateZ 0, because I don't know, and any sain developper would remove it. I suppose this is a hack, but for what ?